source/includes/stackpath/_cdn_settings.md

```diff
@@ -0,0 +1,81 @@
+
+### Retrieve CDN settings
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/cdnsettings/9f236f19-55db-411f-9f05-bd79dc91a69b"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "siteId": "9f236f19-55db-411f-9f05-bd79dc91a69b",
+    "cacheExpirePolicy": "SPECIFY_CDN_TTL",
+    "cacheTtl": 60,
+    "queryStringControl": "CUSTOM",
+    "customCachedQueryStrings": [
+      "customQuery",
+    ],
+    "dynamicCachingByHeaderEnabled": true,
+    "customCachedHeaders": [
+      "mydynamicheader"
+    ],
+    "gzipCompressionEnabled": true,
+    "gzipCompressionLevel": 2,
+    "contentPersistenceEnabled": true,
+    "maximumStaleFileTtl": 30,
+    "varyHeaderEnabled": true,
+    "browserCacheTtl": 60,
+    "corsHeaderEnabled": true,
+    "allowedCorsOrigins": "SPECIFY_ORIGINS",
+    "originsToAllowCors": [
+      "example.org"
+    ],
+    "http2SupportEnabled": true,
+    "http2ServerPushEnabled": true,
+    "linkHeader": "my-link-header",
+    "canonicalHeaderEnabled": true,
+    "canonicalHeader": "my-canonical-header",
+    "urlCachingEnabled": true,
+    "urlCachingTtl": 300
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/cdnsettings/:siteId</a></code>
+
+Retrieve CDN settings for a site in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`siteId`<br/>*UUID*  | A site's unique identifier. 
+`cacheExpirePolicy`<br/>*String* | A site's cache expiry policy. Can be `ORIGIN_CONTROLLED`, `SPECIFY_CDN_TTL`, `NEVER_EXPIRE`, or `DO_NOT_CACHE`. 
+`cacheTtl`<br/>*Integer*  | The time to live for the cache. Depends on the cache expiry policy.
+`queryStringControl`<br/>*String* | The strategy for caching query strings. Can be `IGNORE`, `CACHE_ALL` or `CUSTOM`. 
+`customCachedQueryStrings`<br/>*Array[String]* | List of custom cached query strings. Only visible if the `queryStringControl` attribute is `CUSTOM`
+`dynamicCachingByHeaderEnabled`<br/>*Boolean* | Whether or not to enable dynamic caching by headers.
+`customCachedHeaders`<br/>*Array[String]*  | A list of custom cached headers. Only visible if `dynamicCachingByHeaderEnabled` is enabled. 
+`gzipCompressionEnabled`<br/>*Boolean* | Whether or not to enable gzip compression.
+`gzipCompressionLevel`<br/>*Integer* | The level for the gzip compression. Values are between `1` to `6`. Only visible is `gzipCompressionEnabled` is enabled.
+`contentPersistenceEnabled`<br/>*Boolean* | Whether or not make cached content available after its expiration time.
+`maximumStaleFileTtl`<br/>*Integer*  | The maximum time to live for stale files. Only visible if `contentPersistenceEnabled` is enabled. 
+`varyHeaderEnabled`<br/>*Boolean* | Whether or not to enable honor the vary header in a request.
```

Comment:
is this correct? `to enable honor the vary header` should not be `to enable the vary header`?

Reply:
Yup! I got it from here: 
![image](https://user-images.githubusercontent.com/3630620/111627471-75d2fa00-87c5-11eb-8ffd-c4bfb5256ba3.png)

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/326/commits/a8bb75a6b0a5b5efcfdef89b122363e6f6adde03 did fix small typo here though ahah